kOS/commsTest.ks

```diff
@@ -0,0 +1,40 @@
+//	Testing PEGAS comms system
+@LAZYGLOBAL OFF.
+CLEARSCREEN.
+
+LOCAL requestData IS LEXICON(
+	"type", "request",
+	"data", "liftOffTime",
```

Comment:
To request multiple things, a list of requested data can be provided, e.g. `"data", list("liftOffTime", "launchAzimuth"),`

Reply:
Speaking of it, we will need to come up with some documentation too, if this is to become a fully-formed feature.

Reply:
Done.